guide/interactions/slash-commands.md

```diff
@@ -179,7 +179,40 @@ const data = new SlashCommandBuilder()
 			.setName('server')
 			.setDescription('Info about the server'));
 ```
+### Localizations
 
+::: warning
+You must have [Command-localizations build](https://discord.com/__development/link?s=4bxTIpeKRvD1G1z6Uo%2F%2FKbW1f%2BRrGYb2Klg5NHkLIn0%3D.eyJ0YXJnZXRCdWlsZE92ZXJyaWRlIjp7ImRpc2NvcmRfd2ViIjp7InR5cGUiOiJicmFuY2giLCJpZCI6ImZlYXR1cmUvd2ViLXNsYXNoLWNvbW1hbmQtbG9jYWxpemF0aW9uIn19LCJyZWxlYXNlQ2hhbm5lbCI6bnVsbCwidmFsaWRGb3JVc2VySWRzIjpbXSwiYWxsb3dMb2dnZWRPdXQiOmZhbHNlLCJleHBpcmVzQXQiOiJGcmksIDAzIEp1biAyMDIyIDIxOjE1OjA5IEdNVCJ9) for now
+:::
+Localized names and descriptions depending on the user's selected language.
+
+You can pick localizations from [this list](https://discord.com/developers/docs/reference#locales)
+```js {5-8,10-12,15,16}
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+const data = new SlashCommandBuilder()
+	.setName('dog')
+	.setNameLocalizations({
+		pl: 'pies',
+		de: 'hund',
+	})
+	.setDescription('Get cute picture of dog!')
+	.setDescriptionLocalizations({
+		pl: 'Słodkie zdjęcie pieska!',
+		de: 'Erhalten Sie nettes Bild des Hundes!',
+	})
+	.addStringOption(option => {
+		option.setName('breed').setDescription('Breed of dog')
+		 .setNameLocalizations({
+			 pl: 'rasa',
+			 de: 'rasse',
+			 })
+		 .setDescriptionLocalizations({
+		 pl: 'Rasa psa',
+		 de: 'Rasse des Hundes',
+		 });
+	});
```

Comment:
```suggestion
	.addStringOption(option =>
		option
            .setName('breed')
            .setDescription('Breed of dog')
		    .setNameLocalizations({
			    pl: 'rasa',
			    de: 'rasse',
			})
		    .setDescriptionLocalizations({
		        pl: 'Rasa psa',
		        de: 'Rasse des Hundes',
		    })
	);
```
Maybe it would be better? The sintax in yours is a bit strange.